components/camel-elasticsearch5/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you add nicer copyright header. And also format the XML with only 2 or 4 spaces instead of tabs.